README.rst

```diff
@@ -91,8 +91,9 @@ Starting DALI 0.8.0 for CUDA 10.0 based build execute:
 
    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/cuda/10.0 nvidia-dali-tf-plugin
 
+.. note::
 
-Installing this package will install ``nvidia-dali`` and its dependencies, if these dependencies are not already installed. The package ``tensorflow-gpu`` must be installed before attempting to install ``nvidia-dali-tf-plugin``.
+    Due to `know issue with installing dependent packages <https://github.com/pypa/pip/issues/1386>`_), DALI need to be installed first and then ``nvidia-dali-tf-plugin`` (two separate PIP invocations). The package ``tensorflow-gpu`` must be installed before attempting to install ``nvidia-dali-tf-plugin``.
```

Comment:
Due to a

Reply:
DALI needs to be installed before installing `nvidia-dali-tf-plugin` (in a separate pip install call)

Reply:
Done